firefox_puppeteer/api/permissions.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from ..base import BaseLib
+
+
+class Permissions(BaseLib):
+    def remove(self, host, permission):
+        with self.marionette.using_context('chrome'):
```

Comment:
Libraries should use with_context so they don't make an assumption about the caller's context.